tests/unit/sdk/test_schema.py

```diff
@@ -452,3 +453,8 @@ async def test_display_schema_load_errors_details_when_error_is_in_attribute_or_
   Node: SecurityTailscaleSSHRule | Attribute: check_period (10080) | Extra inputs are not permitted (extra_forbidden)
 """
         assert output == expected_console
+
+
+def test_schema_base__get_schema_name__returns_correct_schema_name_for_protocols():
+    assert InfrahubSchemaBase._get_schema_name(schema=BuiltinTagSync) == "BuiltinTag"
+    assert InfrahubSchemaBase._get_schema_name(schema=BuiltinIPAddressSync) == "BuiltinIPAddress"
```

Comment:
Can you also add the same check for `BuiltinTag` and `BuiltinIPAddress` without the syncs, and additionally also add them as pure strings? I.e. so that we have six asserts?